File: Praktikum 4/c/PRAK405_2410817110012_GtMuhammadLuthfiKamil.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    scanf("%d %d", &n, &kelipatan);

    int total = 0;
    for(int i = 1; i <= n; i++) {
        int sum_result = 0;
        for(int j = i; j >= 1; j--) {
            printf("(%d * %d)", j, kelipatan);
            sum_result += j * kelipatan;
            if(j > 1) {
                printf(" + ");
            }
        }
        total += sum_result;
        printf(" = %d \n", sum_result);
    }

    printf("%d \n", total);
    return 0;
}

File: Praktikum 4/py/PRAK405_2410817110012_GtMuhammadLuthfiKamil.py
def main() :
    n, kelipatan = map(int, input(" ").split())

    total = 0
    for i in range(1, n + 1):
        sum_result = 0
        for j in range(i, 0, -1):
            print(f"({j} * {kelipatan})", end ="")
            sum_result += j * kelipatan
            if j > 1:
                print(" + ", end="")
        print(f" = {sum_result}")
        total += sum_result
           
    print(total)

if __name__ == "__main__":
    main()      